client/src/routes/Menu/index.jsx

```diff
@@ -32,11 +32,18 @@ const {
   emptySmallEllipse,
   fullSmallEllipse,
   marginLeft15px,
+  img,
+  imgWrapper,
+  companyName,
+  companyWrapper,
 } = styles;
 
 function Menu() {
   return (
     <Container>
+      <div className={imgWrapper}>
+        <img src="./assets/logo.svg" alt="guy in vr" className={img} />
```

Comment:
The logo should be an object.

The logo has the wrong alt text.

Reply:
Alt Text has been updated. 

We will have to use a direct link for now has using an object doesn't generate he log.